FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/FlowcryptApiRepository.kt

```diff
@@ -54,17 +54,16 @@ import java.util.concurrent.TimeUnit
 class FlowcryptApiRepository : ApiRepository {
```

Comment:
What's the difference between `FlowcryptApiRepository` and `ApiRepository` ?

Reply:
Please look at https://github.com/FlowCrypt/flowcrypt-android/pull/2171#discussion_r1086608449